docs/EXTENDED-ARCHIVES.md

```diff
@@ -75,28 +75,32 @@ Archive layout
 The contents of an extended archive are the followings:
 
 - **REQUIRED** `.meta/manifest.json`
-- `.meta/schema.json`
-- `.meta/instructions.markdown`
-- `.meta/release-notes.txt`
 - **REQUIRED** `<image name>.<extension>`
 - `.meta/checksum`
 
 ### Order
 
 Notice that the order is important. The `.meta/` directory must come before the
-image data, since we need that information before being able to make use of the
-image.
-
-The order of the files inside `.meta/` is not mandatory, however the order
-described above ensures the maximum UX perceived efficiency.
+image data, and `.meta/manifest.json` should be the first entry, since we need
+that information before being able to make use of the image.
 
 ### Graphics
 
 Any graphic included in the archive can be either an [`svg`][svg] file, or a
-[`png`][png] file with the `<filename>.png` and `<filename>@2x.png` convention.
+[`png`][png] file.
 
 If both are present, [`svg`][svg] should be favoured by the client.
 
+Ifa [`png`][png] file path is declared and the client detects that the host is
+running a high definition screen, it may search for a `<filename>@2x.png` file
+in the archive.
+
+### Paths
+
+The `.meta/manifest.json` file may refer to other files in the package. If the
+property searches for the file relative to `.meta`, then it should not allow
+accessing the parent directory with `..`.
```

Comment:
Probably easier to just rephrase this as filenames can't include any path segments, i.e. no `/`